regolith/schemas.py

```diff
@@ -3242,21 +3244,16 @@
         },
         "type": {
             "description": "type of presentation",
-            "eallowed": [
-                "award",
-                "colloquium",
-                "contributed_oral",
-                "invited",
-                "keynote",
-                "plenary",
-                "poster",
-                "seminar",
-                "tutorial",
-                "webinar"
-            ],
+            "eallowed": PRESENTATIONS_TYPE,
             "required": True,
             "type": "string",
         },
+        "webinar": {
+            "description": "true if a webinar",
+            "required": False,
+            "type": "boolean",
+            "default": False,
```

Comment:
does this work, setting default in the schema?  Actually, I don't know the answer, this is not a rhetorical question.....

Reply:
I am not sure, but I think I do not even need this.
 In the code, I am only adding the webinar if it is true, thus it is not necessary to set it to false by default, bc if it is none or false, it will do nothing. Or should I guarantee that when there is nothing in the 'webinar' field it should be defined as false?

Reply:
no, you are right, just remove this.